### ISSUE TITLE
Add support for TailwindCSS 4.0 @plugin and friends

### ISSUE THREAD

**Issue by mhenrixon** [SublimeText/TailwindCSS#9]
TailwindCSS 4.0 adds a few new keywords to move the entire configuration to CSS and to speed up compilation considerably. 

These are the current ones that I have discovered that needs to be added. I am guessing it goes in this package and not the CSS package.

```css
@plugin "daisyui";
@plugin "daisyui/theme" {
  name: "light";
  default: true;
  --color-primary: red;
}

@source "./app/views/**/*.rb";
@source "./app/components/**/*.rb";
@source "./app/**/*.html.erb";
@source "./app/helpers/**/*.rb";
@source "./app/javascript/**/*.js";
@source "./app/assets/stylesheets/**/*.css";
@source "./node_modules/daisyui/dist/**/*.css";
@source "./node_modules/daisyui/dist/**/*.js";
```

**Comment by deathaxe** [comment]
Resolved by v1.5.0

**Comment by mhenrixon** [comment]
> Resolved by v1.5.0

Thanks a bunch @deathaxe that was blazing fast!!

**Comment by deathaxe** [comment]
It was well prepared :)
